Makefile

```diff
@@ -569,7 +569,7 @@ remoteintegration: test-binaries ginkgo-remote
 
 .PHONY: localbenchmarks
 localbenchmarks: test-binaries
-	ACK_GINKGO_RC=true $(GOBIN)/ginkgo \
+	PATH=$(PATH):$(shell PWD)/hack ACK_GINKGO_RC=true $(GOBIN)/ginkgo \
```

Comment:
Ah, sorry, should've looked earlier. `shell PWD` is going to run the command `PWD`, which doesn't exist, and will fail quickly. It needs to be lower-case `shell pwd`.

Reply:
@vrothberg  ICYMI in the kerfuffle: this needs to be lower-case `pwd`